packages/google_maps_flutter/google_maps_flutter_android/example/integration_test/google_maps_tests.dart

```diff
@@ -1503,6 +1503,7 @@ Marker _copyMarkerWithClusterManagerId(
     rotation: marker.rotation,
     visible: marker.visible,
     zIndex: marker.zIndex,
+    zIndexInt: marker.zIndexInt,
```

Comment:
This won't be needed after the suggested change in the platform interface. The `zIndex` will just need a comment saying that the deprecated parameter is being used to avoid losing precision.